sjsonnet/src/sjsonnet/Importer.scala

```diff
@@ -43,8 +43,8 @@ class CachedImporter(parent: Importer) extends Importer {
 
 class CachedResolver(
   parentImporter: Importer,
-  val parseCache: mutable.HashMap[(Path, String), Either[Error, (Expr, FileScope)]] = new mutable.HashMap
-) extends CachedImporter(parentImporter) {
+  val parseCache: ParseCache = new DefaultParseCache
```

Comment:
Let's remove this default value so we can make sure we pass the right parse cache from all code paths.

This means you'll have to find all places where this method is called and decide if it should pass a fresh `DefaultParseCache` (if it's on a code path that's NOT coming from Bazel, meaning mostly tests) or make sure we have an external parse cache.